### DISCUSSION TITLE
能精確生成文字圖片: Ideogram

### DISCUSSION THREAD

**Discussion by OG-Matcha** [1101-GS4719-C/Python-Community-GS4719-C#1909] [科技新知]
我自己有使用過蠻多種文生圖的軟體，但是大部分都較難以在圖片中精確地生成文字。

而 [Ideogram](https://ideogram.ai) 的亮點就在於能夠在「圖片中」精確「生成文字」。

![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/85878531/9585396b-2c75-4e7d-9f13-86d2ae9d113d)

在 2/28 時， Ideogram 正式推出了 Ideogram 1.0 模型，大幅提升了渲染文字的準確性。根據官方數據，在渲染文字圖片上比 OpenAI DALL·E 3 錯誤率還低了 2 倍。

另外， Ideogram 1.0 也加入目前主流 AI 圖片生成模型的功能，例如：Magic Prompt，只需要簡單的一句話，就能自動幫你寫出完整的 Prompt。

我自己嘗試過是可以很好的符合需求，我覺得相較於其他生成應用，或許當有需要在圖片中嵌入文字的話是相當好用，就算是免費方案也有 100 images/day (25 prompts/day) 的資源。

**Comment by sunset513** [comment]
我覺得很實用耶，我在做圖文創作的話會更有效率，感謝分享

**Comment by benbebobo149** [comment]
對於文章中提到的 Ideogram，我深感興趣。我曾嘗試過許多文生成圖軟體，但很少能在圖片中準確生成文字。 Ideogram 的能力令我印象深刻，尤其是它在渲染文字方面的準確性。但我更推薦 krea.AI，因為它的功能也相當酷，特別是它能將 Logo 融入圖片中，營造出奇特的視覺效果。這種創新性讓我感到振奮，同時也讓我更期待探索這些 AI 技術的未來發展。
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/61257062/d24bd944-dedc-40b5-969e-0b0d39e3cf9b)
圖為官網給的一個範例。

**Comment by SusuTima** [comment]
是一個實用的好工具，我去Ideogram打上「cat with snow」，以下為生成結果:
![cat_with_snow](https://github.com/NCU-GS4719-Python/Python-Community/assets/132805404/8f904e1a-e77d-4b54-979b-8095ec373934)
得到一隻雪白的小貓咪

**Comment by SusuTima** [comment]
記得之前也看過一本介紹AI繪圖的書，叫「未來藝世界：AI 繪圖新旅程」，裡面有各種類型的AI繪圖，帶我用AI的視角去看AI描繪的藝術。

**Comment by YifenLin** [comment]
之前有用過一款免費圖文生成網站Microsoft designer, 現在又多了一個選擇，之後可以玩玩看Ideogram,然後比較兩者優異。

**Comment by wanderer657** [comment]
最近AI應用的興起，讓各種版面設計軟體蓬勃發展，我認為可以混合使用不同的軟體，例如隔壁篇的microsoft designer，可以讓版面設計更加豐富多彩，不僅止於ppt 內部的傳統版面配置工具。

**Comment by wanka2030** [comment]
Photoshop裡面的AI圖片生成雖然已經鰻好用的，但如果能在上面增添文字也可以讓生成出來的圖片更有吸引力。不單單只是圖片生成，現在的SORA影片產生也是值得大家注目的新勢力，未來也只會有更多的AI圖片生成供大家使用:D也謝謝好工具推推
